src/components/HoverableDiv/HoverableDiv.js

```diff
@@ -1,19 +1,20 @@
 import React from 'react'
-import HoverVideo from './HoverVideo'
+import HoverVideo from './HoverVideo';
```

Comment:
Since you moved the css to 'HoverableDiv.css', you should now write `import './HoverableDiv.css'` in order for the CSS to work for this file